_includes/release-notes/v3.13.0-release-notes.md

```diff
@@ -27,3 +27,7 @@ If that's whetted your appetite and you'd like to hear more (and see some pretty
  - In calico.yaml, `CALICO_IPV4POOL_CIDR` has been commented out, but the default CIDR remains the same. To change the CIDR in the manifest, you must first uncomment that section. [calico #3211](https://github.com/projectcalico/calico/pull/3211) (@rafaelvanoni)
  - Improve Felix liveness reporting when handling large policies. [felix #2215](https://github.com/projectcalico/felix/pull/2215) (@fasaxc)
  - Improve IPAM garbage collection for etcd clusters. [kube-controllers #459](https://github.com/projectcalico/kube-controllers/pull/459) (@caseydavenport)
+
+#### Compatibility
+
+Calico v.3.13.0 has been tested against Kubernetes v1.15, v1.16, and v1.17.
```

Comment:
typo: v.3.13

Reply:
We could add this to the auto-generation template as well - might be a good idea:

https://github.com/projectcalico/calico/blob/master/release-scripts/generate-release-notes.py#L88